hummingbot/connector/connector_status.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+def get_connector_status(connector_name: str) -> str:
+    """
+    Indicator whether a connector is working properly or not. Unknown means the connector is not in the dict.
+    Unavailable means a connector doesn't work, Warning means the connector works but has issues.
+    Ok means a connector is working properly. The connector_status dict is updated every new release.
+    """
+    connector_status = {
```

Comment:
This list should be outside of the function, simply as a member of this module (file).
The reason is the list will get updated often, if it is in the function, technically we'll need to retest the function and it's easier to maintain and update.

Reply:
Thanks Jack, I just pushed a commit with this change.